README.md

```diff
@@ -278,6 +279,36 @@ The answers to those questions are based on my humble knowledge. If you find som
 
 2. TDD. What is TDD?
 
+## ARCHITECTURE
```

Comment:
@ducminhgd Thank you for your PR. About your modification:
- I see some questions don't belong to the topic ARCHITECTURE and/or only one technology specified (this repo dedicates to general, not particular questions), like questions no 10, 11, 14, 16, 19, 10, 22, 23, 24.
- Talking about architecture, I think it would be better if we divide it into sub-topics, like:
1. Monolith:
Q1
Q2
...
2. Microservices:
Q1
Q2
...
- It would be better if you can rephrase the answer in your own words, it will help you remember these things much much longer.

Reply:
Hi @tamhoang1412 , since I literally copied those question from an articles, I think it is better if I keep those questions and answers as they are in original article.

If you still keep you opinion then I will close this pull request and create a new one, which is re-phrased.

Reply:
> this repo dedicates to general, not particular questions

I wonder what "particular questions" you meant. We should not talk about a specific technology? For example, is it OK if I contribute a question such as "What is GIL in Python?"?